src/dotnet/APIView/APIViewWeb/Languages/CppLanguageService.cs

```diff
@@ -315,30 +318,40 @@ string GenerateUniqueMethodId(CppAstNode methodNode)
                 return bldr.ToString();
             }
 
-            NavigationItem ProcessClassNode(CppAstNode node, string parentName)
+            NavigationItem ProcessClassNode(CppAstNode node, string parentName, List<CppAstNode> templateParams = null)
             {
                 NavigationItem navigationItem = null;
 
-                // Skip empty forward declarations
-                if (node.inner == null)
```

Comment:
I think there was a specific ask to disable forward declaration.

Reply:
IIRC, the problem is that the ClassTemplateDecl had a CxxRecordDecl with no child nodes, which meant that template specializations without fields wouldn't display.